### ISSUE TITLE
ToggleButtons description width doesn't change with `style` kwarg

### ISSUE THREAD

**Issue by hinnefe2** [jupyter-widgets/ipywidgets#1691]
Following the docs [here](https://ipywidgets.readthedocs.io/en/latest/examples/Widget%20Styling.html#Description) I'm trying to change the width of the description of a `ToggleButtons` by passing in a `style` kwarg. This works for an `IntSlider` as in the docs, but not for a `ToggleButtons`.

If someone can point me towards the relevant parts of `IntSlider` and `ToggleButtons` I'm happy to tag a stab at fixing this myself.

MWE:

```
from ipywidgets import IntSlider, ToggleButtons
from IPython.display import display

style = {'description_width': 'initial'}

# description reads 'A too long ...'
display(IntSlider(description='A too long description'))

# description reads 'A too long description'
display(IntSlider(description='A too long description', style=style))  

 # description reads 'A too long ...'
display(ToggleButtons(description='A too long description', options=['a','b'], style=style)) 
```

Versions:
```
{'commit_hash': 'd86648c5d',
 'commit_source': 'installation',
 'default_encoding': 'UTF-8',
 'ipython_path': '/Users/hhinnefeld/miniconda3/lib/python3.5/site-packages/IPython',
 'ipython_version': '6.1.0',
 'os_name': 'posix',
 'platform': 'Darwin-16.6.0-x86_64-i386-64bit',
 'sys_executable': '/Users/hhinnefeld/miniconda3/bin/python',
 'sys_platform': 'darwin',
 'sys_version': '3.5.2 |Continuum Analytics, Inc.| (default, Jul  2 2016, '
                '17:52:12) \n'
                '[GCC 4.2.1 Compatible Apple LLVM 4.2 (clang-425.0.28)]'}

ipykernel==4.6.1
ipython==6.1.0
ipython-genutils==0.2.0
ipywidgets==7.0.0
jupyter-client==5.1.0
jupyter-console==5.1.0
jupyter-core==4.3.0

```

**Comment by jasongrout** [comment]
Thanks! It looks like in https://github.com/jupyter-widgets/ipywidgets/blob/1bc4a6765565fe39290c4007a3c77a4639bf3c75/packages/controls/src/widget_selection.ts#L363, ToggleButtonsStyle should inherit from DescriptionStyleModel, just like the slider does at https://github.com/jupyter-widgets/ipywidgets/blob/1bc4a6765565fe39290c4007a3c77a4639bf3c75/packages/controls/src/widget_int.ts#L51. It looks like this may be a one-line fix.

**Comment by jasongrout** [comment]
It looks like the python side needs to be changed to: https://github.com/jupyter-widgets/ipywidgets/blob/1bc4a6765565fe39290c4007a3c77a4639bf3c75/ipywidgets/widgets/widget_selection.py#L388 needs to inherit like https://github.com/jupyter-widgets/ipywidgets/blob/1bc4a6765565fe39290c4007a3c77a4639bf3c75/ipywidgets/widgets/widget_int.py#L147

**Comment by jasongrout** [comment]
@hinnefe2 - did you want to tackle this?

**Comment by hinnefe2** [comment]
Hey yeah, sorry to leave it hanging, I'll have a go at it tonight.

**Comment by jasongrout** [comment]
Let us know if you have any questions. I'd like to merge this for 7.0.1 in the next day or two.

**Comment by jasongrout** [comment]
Fixed by #1714.
